### ISSUE TITLE
docs.gitops.weave.works/ seems to have lost the DNS

### ISSUE THREAD

**Issue by Bengreen** [weaveworks/weave-gitops#4235]
**Describe the bug**
the docs site and other sides off weave.works seem to be broken.
Looks like someone else has taken over the domain

**Environment**

web

**To Reproduce**

Browse to : [docs.gitops.weave.works/](https://docs.gitops.weave.works/)

**Expected behavior**

View dos

**Actual Behavior**

Views an advertisement website

**Additional Context (screenshots, logs, etc)**

<img width="880" alt="image" src="https://github.com/user-attachments/assets/0cbf0750-6f1f-455c-bbb5-ecca85d39b97">

**Comment by kingdonb** [comment]
That's correct, the domain is expired. We were told someone would likely buy it, but right now it looks like the registrar has it parked (eg. they're holding it for ransom)

AIUI someone has registered weaveworks.org for the effort to revive Weave GitOps, and ideally some time in the future, `weave.works` will be returned to the unregistered pool and those requests will start to return `NXDOMAIN`

If there are any dead links you're worried about, I was able to find all of the content that we ever referenced on fluxcd.io and fix it with links to web.archive.org. Meanwhile if you're trying to access the docs, I started taking steps to re-host them here:

https://github.com/microscaler/gitops-docs

There is no live website right now as far as I know.

**Comment by kingdonb** [comment]
Ref:

* https://github.com/fluxcd/website/pull/2103

**Comment by casibbald** [comment]
we will soon move to weaveworks.org

A pre-release supporting Flux v2.4.0 is in QA and if all goes well we may get it out before Christmas!

**Comment by casibbald** [comment]
Docs now at https://docs.gitops.weaveworks.org
This link will be in the new release.
